src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,25 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 
+@Entity
 public class Movie implements Cloneable {
```

Comment:
how are your tables called in the DB?
maybe we should add the `@Table` annotation for all entities?